02LessonTwo/homework/index.html

```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>BRIGHT STAR TELECOMMUNICATIONS</title>
+  <title>ELSA'S BIO</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-  <img src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcQ_36cdRf-KbdFiziParG_J7jNVgWOFcGJ33OtLBEr9DMkeG-NxMg" alt="">
-<h1 style="color: gold">Bright Star Telecommunictions</h1>
+  <img src="IMAGES/ELSA_PIC1.jpg" alt="">
```

Comment:
Great thought creating a separate folder for images; helps keep things organized.